data/cset.csv

```diff
@@ -225,4 +225,5 @@
 2.16.840.1.113762.1.4.1247.71,1000000232,https://docs.google.com/spreadsheets/d/1jzGrVELQz5L4B_-DqPflPIcpBaTfJOUTrVJT5nS_j18/edit#gid=612526229,VSAC,,
 2.16.840.1.113762.1.4.1247.86,1000000233,https://docs.google.com/spreadsheets/d/1jzGrVELQz5L4B_-DqPflPIcpBaTfJOUTrVJT5nS_j18/edit#gid=612526229,VSAC,,
 2.16.840.1.113762.1.4.1247.70,1000000234,https://docs.google.com/spreadsheets/d/1jzGrVELQz5L4B_-DqPflPIcpBaTfJOUTrVJT5nS_j18/edit#gid=612526229,VSAC,,
-2.16.840.1.113762.1.4.1196.788,1000000235,https://docs.google.com/spreadsheets/d/1jzGrVELQz5L4B_-DqPflPIcpBaTfJOUTrVJT5nS_j18/edit#gid=612526229,VSAC,,
```

Comment:
@Sigfried Hmm, interesting. I only added one OID to the end of this file. It looks like it's saying here that there are only a couple lines that are different. However, when I `git diff`, it shows that all the lines are different, and that there are `^M` (a line break character) at the end of all my lines. I guess this is something that Excel did.

Looks like this will be a pretty easy merge, then.